forgettable-frontend/src/hoc/PageRouter/PageRouter.js

```diff
@@ -41,6 +44,10 @@ const PageRouter = (props) => {
     <div className="page-wrapper">
       <Routes>
         <Route path="/" element={<Home />} />
+        <Route path="company" element={<CompanyListPage/>} />
```

Comment:
Can you nest these routes instead of using company/ for the next three routes if possible?

Reply:
i have tried to nest the routes like this:
`<Match pattern="/" component={Frontpage}> 
  <Match pattern="/home" component={HomePage} />
  <Match pattern="/about" component={AboutPage} />
</Match>`

Seems that In react-router-v4 you don't nest <Routes />. Instead, you put them inside another <Component /> which i am not sure if that is needed since the previous group has not put them inside the component